forum/migration_helpers.py

```diff
@@ -251,12 +254,20 @@ def migrate_read_states(db: Database[dict[str, Any]], course_id: str) -> None:
                         mongo_id=thread_id
                     ).content_object_id
                     thread = CommentThread.objects.filter(id=thread_id).first()
-                    if thread:
-                        LastReadTime.objects.get_or_create(
+                    if not thread:
```

Comment:
Does this code change anything? I find it much more confusing than the previous version (too many `if`)

Reply:
this change was necessary to resolve CI check because it was increasing the nested blocks numbers. So i just added the early return statement to reduce the nested number and didn't added another if/else in the if thread condition.

In LastReadTime, read_state and comment_thread should be unique together due to which code was throwing an error while updating the timestamp. That's why these changes were needed.